src/jit/Backend.cpp

```diff
@@ -784,6 +784,7 @@ static void emitBrTable(sljit_compiler* compiler, BrTableInstruction* instr)
     if (targetLabelCount <= JITCompiler::kMaxInlinedBranchTable) {
         CompileContext* context = CompileContext::get(compiler);
         JITCompiler::BranchTableLabels* brTableLabels = new JITCompiler::BranchTableLabels(context->compiler, targetLabelCount);
+        context->compiler->appendBrTableLabel(brTableLabels);
```

Comment:
These are already in a chain list:
https://github.com/Samsung/walrus/blob/main/src/jit/Compiler.h#L751
You can just delete the chain.

Reply:
You are right, I did not look into it deeply enough. I believe it should be correct now.